core-java-modules/core-java-collections-6/src/test/java/com/baeldung/nullcheck/NullValueCheckUnitTest.java

```diff
@@ -0,0 +1,92 @@
+package com.baeldung.nullcheck;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.TreeSet;
+import java.util.function.Function;
+
+import org.junit.jupiter.api.Test;
+
+class NullValueCheckUnitTest {
+
+    Function<Collection<?>, Long> countNulls = collection -> collection.stream()
+        .filter(Objects::isNull)
+        .count();
+
+    @Test
+    void givenList_whenNullValueAdded_doesNotFail() {
+        // adding nulls to a list
+        Integer[] numberArray = { null, 0, 1, null, 2, 3, null };
+        List<Integer> numbers = Arrays.asList(numberArray);
+        assertEquals(countNulls.apply(numbers), 3);
```

Comment:
Wrong order, please swap these 2 arguments: expected and actual value.

Reply:
Fixed.